source/board/m48ssidae_bl.c

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file    m48ssidae_bl.c
+ * @brief   board ID and meta-data for the hardware interface circuit (HIC) based on Nuvoton M48SSIDAE
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2009-2019, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_config.h"
+#include "daplink_addr.h"
```

Comment:
Build fails - linker error for M48SSIDAE_NUMAKER_M252KG_IF: 

line 22: Error:  #5: cannot open source input file "daplink_addr.h": No such file or directory

Reply:
I have been fixed the include path for make_armcc. Could you check it again?

Reply:
I have fixed the include path for make_armcc. Could you check it again?

Reply:
CI restarted